src/signalrclient/web_request.h

```diff
@@ -26,7 +25,6 @@ namespace signalr
     private:
         const web::uri m_url;
         web::http::http_request m_request;
-        utility::string_t m_user_agent_string;

```

Comment:
I think you need to fix [this test](https://github.com/SignalR/SignalR-Client-Cpp/blob/c152e53fc27d4a7670baa3a7953b99edeab88be5/test/signalrclienttests/web_request_tests.cpp#L30) since you removed this method.

Reply:
(I meant to put this comment on this line: https://github.com/SignalR/SignalR-Client-Cpp/pull/171/files/4c93b793d7617af9f89eb862e3312a04097cc067#diff-5c7754f6ce7e6dd559ff3760f615595cL17)

Reply:
I'll check it, overlooked the tests